src/main/java/com/google/api/generator/gapic/composer/ResourceNameHelperClassComposer.java

```diff
@@ -96,4 +204,61 @@ public GapicClass generate(ResourceName resourceName) {
                 c -> c.getSimpleName(),
                 c -> TypeNode.withReference(ConcreteReference.withClazz(c))));
   }
+
+  private static Map<String, VariableExpr> createFixedClassMemberVariables() {
+    Map<String, TypeNode> memberVars = new HashMap<>();
```

Comment:
Curious the reason why we call it `fixedClassMemberVariable`, seems it has String as key and its corresponding TypeNode as value. Will we append more key-values here? Thanks! 
And some comments would be helpful for this helper if possible

Reply:
The "fixed" part refers to the independent nature of these variables, in the sense that they don't depend on the underlying protobuf.